src/TestInfrastructure/Framework/ConfigurableVsProjectSystemHelper.cs

```diff
@@ -82,6 +82,28 @@ IVsHierarchy IProjectSystemHelper.GetIVsHierarchy(Project dteProject)
             return dteProject as IVsHierarchy;
         }
 
+        public IEnumerable<Project> GetSelectedProjects()
+        {
+            return this.SelectedProjects;
+        }
+
+#pragma warning disable S3215 // "interface" instances should not be cast to concrete types
+        public string GetProjectProperty(Project dteProject, string propertyName)
+        {
+            return (dteProject as ProjectMock)?.GetBuildProperty(propertyName);
```

Comment:
Assert.Inconclusive in all those cases with a message saying that only ProjectMock is expected